chromadb/test/segment/distributed/test_memberlist_provider.py

```diff
@@ -1,123 +0,0 @@
-# Tests the CustomResourceMemberlist provider
```

Comment:
I think this test is worth keeping around because we don't have equivalent coverage elsewhere maybe